.github/linters/.hadolint.yaml

```diff
@@ -0,0 +1,4 @@
+---
+ignored:
```

Comment:
Why ignore these? DL3022 is quite useful, for example.

Reply:
Probably a sloppy shortcut when I was trying to avoid making any changes that would break code, I'll take another look at this one